build/Build.Pack.cs

```diff
@@ -204,6 +213,10 @@ void BuildMsiInstallerForPlatform(MSBuildTargetPlatform platform, AbsolutePath w
                     wixProjectFile.ReplaceTextInFile("{WixTargetsPath}", wixNugetPackagePath / "tools" / "Wix.targets");
                     wixProjectFile.ReplaceTextInFile("{WixTasksPath}", wixNugetPackagePath / "tools" / "wixtasks.dll");
 
+                    var tentacleInstallerWixProduct = RootDirectory / "installer" / "Octopus.Tentacle.Installer" / "Product.wxs";
```

Comment:
I have a faint memory that you can pass arguments into Wix when building it.

If this is true, would that be nicer than manipulating the file?

Reply:
If you know how, yep, I was just following our existing pattern

Reply:
Although, it does look like we already manipulate the files...

Reply:
Maybe a Sharpening exercise 😁